app/src/components/statusBar/statusBar.scss

```
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.status-bar {
+  display: flex;
+  align-items: center;
+  gap: 2px;
+
+  .count {
+    margin-right: 10px;
+  }
+  .status-line {
```

```suggestion
  }

  .status-line {
```

Let's split rules by empty line, please update all places in the PR